NEWS

```diff
@@ -12,6 +12,14 @@
       o Allow GNU style "make variables" to be used with Configure.
       o Add a STORE module (OSSL_STORE)
       o Claim the namespaces OSSL and OPENSSL, represented as symbol prefixes
+      o Add multi-prime RSA (RFC 8017) support
+      o Add SM3 implemented according to GB/T 32905-2016
+      o Add SM4 implemented according to GB/T 32907-2016.
+      o Add 'Maximum Fragment Length' TLS extension negotiation and support
+      o Add ARIA support
+      o Add SHA3
+      o Rewrite of devcrypto engine
+      o Add support for SipHash
 
```

Comment:
@mattcaswell, if it's not too late yet, here is a raw draft for NEWS about the RNG. Probably it's too verbose, and probably @richsalz should have a look over it.

- Replace the default RAND method by new implementation using a CTR-DRBG (Deterministic Random Bit Generator based on AES-CTR) according to NIST SP800-90Ar1. It is essentially a port of the default RAND method from the FIPS 2.0 Object Module.
- Keep the DRBG state in secure storage, if it is enabled.
- Added seed chaining of DRBG instances. Every SSL object now has its own DRBG instance, to reduce lock contention and increase unpredictability of random output.
- Use a dedicated DRBG for generating long term private keys.


_TODO:_ Is there already an entry about secure storage? I didn't see anything.

Reply:
- The DRBG supports automatic seeding and reseeding without depending on support from the applicaion.